packages/@aws-cdk/aws-cloudfront/test/integ.distribution-lambda.expected.json

```diff
@@ -70,9 +70,7 @@
       "Properties": {
         "DistributionConfig": {
           "DefaultCacheBehavior": {
-            "ForwardedValues": {
-              "QueryString": false
-            },
+            "CachePolicyId": "658327ea-f89d-4fab-a63d-7e88639e58f6",
```

Comment:
Maybe worth it to have a non-default value tested in one of these integ tests, since we're changing them anyway...?

Reply:
Yup, makes sense.